src/Campaigns/resources/admin/components/CampaignDetailsPage/index.tsx

```diff
@@ -172,6 +177,14 @@ export default function CampaignsDetailsPage({campaignId}) {
         })();
     };
 
+    const getCampaignPageUrl = (campaignPage: { id: any; slug: any; link: any; }) => {
```

Comment:
Let's avoid using the `any` type when we can as its not very helpful.  Also I believe this whole function can be moved outside of the component so its not re-created between re-renders.

```suggestion
    const getCampaignPageUrl = (campaignPage: { id: number; slug: string; link: string; }) => {
```